seek-n-major.json

```diff
@@ -0,0 +1,135 @@
+        "commitMessageExtra": "",
+        "groupName": "gomod digests",
+        "schedule": "after 3am and before 6am on the first day of the month"
+      },
+      "matchManagers": ["gomod"],
+      "semanticCommitType": "fix"
+    },
+    {
+      "commitMessageExtra": "{{newValue}}",
+      "commitMessageTopic": "{{depName}}",
+      "matchManagers": ["buildkite", "gomod", "npm", "nvm"]
+    },
+    {
+      "matchManagers": ["npm"],
+      "matchPackageNames": ["@types/node"],
+      "matchUpdateTypes": ["major"],
+      "enabled": false
+    },
+    {
+      "automerge": true,
+      "commitMessageExtra": "",
+      "groupName": "npm definitely typed",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@types/"],
+      "prPriority": 99,
+      "recreateClosed": true,
+      "matchUpdateTypes": ["major"],
+      "schedule": "before 3am on Tuesday"
+    },
+    {
+      "commitMessageExtra": "",
+      "depTypeList": ["devDependencies"],
+      "excludePackageNames": ["braid-design-system", "sku", "skuba"],
+      "excludePackagePatterns": ["^@?seek", "seek$", "^@types/"],
+      "groupName": "npm dev dependencies",
+      "matchManagers": ["npm"],
+      "recreateClosed": true,
+      "matchUpdateTypes": ["major"],
+      "schedule": "after 3am and before 6am on Tuesday"
+    },
+    {
+      "commitMessageExtra": "",
+      "depTypeList": ["peerDependencies"],
+      "excludePackageNames": ["braid-design-system", "sku", "skuba"],
+      "excludePackagePatterns": ["^@?seek", "seek$", "^@types/"],
+      "groupName": "npm peer dependencies",
+      "matchManagers": ["npm"],
+      "recreateClosed": true,
+      "matchUpdateTypes": ["major"],
+      "schedule": "after 3am and before 6am on Tuesday"
+    },
+    {
+      "commitMessageExtra": "",
+      "excludePackagePatterns": ["^seek-jobs/", "^seek-oss/"],
+      "groupName": "buildkite plugins",
+      "managerBranchPrefix": "",
+      "matchManagers": ["buildkite"],
+      "recreateClosed": true,
+      "schedule": "after 3am and before 6am on Wednesday"
+    },
+    {
+      "commitMessageExtra": "",
+      "group": {
+        "commitMessageTopic": "{{groupName}}"
+      },
+      "groupName": "docker images",
+      "matchManagers": ["docker-compose", "dockerfile"],
+      "recreateClosed": true,
+      "schedule": "after 3am and before 6am on Wednesday"
+    },
+    {
+      "matchPackageNames": ["braid-design-system", "sku", "skuba"],
+      "matchPackagePatterns": ["^@?seek", "seek$"],
+      "prPriority": 98,
+      "schedule": "after 3am and before 6am on every weekday"
+    },
+    {
+      "commitMessageExtra": "",
+      "depTypeList": ["dependencies"],
+      "groupName": "aws-sdk",
+      "matchManagers": ["npm"],
+      "matchPackageNames": ["aws-sdk"],
+      "matchPackagePatterns": ["^@aws-sdk/"],
+      "recreateClosed": true,
+      "matchUpdateTypes": ["major"],
+      "schedule": "after 3am and before 6am on the first day of the month"
+    },
```

Comment:
We don't need to special case this here; the AWS SDK is only noisy when you're dealing with minor version upgrades.

```suggestion
```